{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -15,8 +15,13 @@ PYTHON_INTERPRETER = python
 .PHONY: requirements
 requirements:
 	{% if "requirements.txt" == cookiecutter.dependency_file -%}
+	{% if "uv" == cookiecutter.environment_manager -%}
+	uv pip install pip
```

Comment:
I don't think we need to install pip if uv is being chosen as the environment manager.